R/create_output_type_item.R

```diff
@@ -70,30 +82,39 @@ create_output_type_point <- function(output_type = c("mean", "median"),
     ))
   }
   output_type <- rlang::arg_match(output_type)
-  # Get output type id property according to config schema version
-  # TODO: remove back-compatibility with schema versions < v2.0.0 when support
-  # retired
-  config_tid <- hubUtils::get_config_tid(
-    config_version = hubUtils::get_schema_version_latest(schema_version, branch)
-  )
-
```

Comment:
The above is only relevant to older schema versions so just moved it to `pre_v4` section